etc/neutron/services/f5/f5-openstack-agent.ini

```diff
@@ -425,8 +425,7 @@ f5_common_external_networks = True
 # for all mapped fixed_ips will be made referencing the ports id
 # and the ports device_id.
 #
-# l3_binding_static_mappings = 'subnet_a':[('port_a','device_a'),('port_b','device_b')], 'subnet_b':[('port_c','device_a'),('port_d','device_b')]
-#
+# l3_binding_static_mappings = {"subnet_b": [["port_c", "device_a"], ["port_d", "device_b"]], "subnet_a": [["port_a", "device_a"], ["port_b", "device_b"]]}
```

Comment:
Yes, the other line is not parseable.  If we provide an example, it should work with the code.